src/traces/scatter/attributes.js

```diff
@@ -65,6 +65,10 @@ module.exports = {
             'See `y0` for more info.'
         ].join(' ')
     },
+    key: {
+        valType: 'data_array',
+        description: 'A list of keys for object constancy of data points during animation'
```

Comment:
Let's remove `key` from the attributes for now.

Looks like it is only used internally at the moment (it is not getting coerced anywhere, right?).

I wouldn't want to pollute our attribute [schema](https://plot.ly/javascript/reference/) before implementing the functionality.